app/frontend/src/components/Team.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <v-container>
+    <p class="text-right">
+      <strong>
+        <v-icon class="pr-2">group</v-icon>Team members
+        <span v-if="usersLoaded">({{users.length}})</span>
+        <v-tooltip bottom open-delay="1000">
```

Comment:
There a reason why we're doing a delay of 1 second as opposed to immediate?
Consideration - would the tooltip look better if it was top instead of bottom?

Reply:
Immediate looked annoying with it popping up as soon as you go to click.
Will consult with Mina about these when we do more UI checking later.